app/src/main/java/com/sun/moviedb/screen/login/LoginActivity.kt

```diff
@@ -0,0 +1,80 @@
+package com.sun.moviedb.screen.login
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.LinearLayout
+import android.widget.Toast
+import androidx.activity.enableEdgeToEdge
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.ViewCompat
+import androidx.core.view.WindowInsetsCompat
+import com.google.android.gms.auth.api.signin.GoogleSignIn
+import com.google.android.gms.auth.api.signin.GoogleSignInClient
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions
+import com.google.android.gms.common.api.ApiException
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.GoogleAuthProvider
+import com.sun.moviedb.R
+import com.sun.moviedb.screen.MainActivity
+
+class LoginActivity : AppCompatActivity() {
+
+    var loginButton: LinearLayout? = null
+
+    private lateinit var auth: FirebaseAuth
+    private lateinit var googleSignInClient: GoogleSignInClient
```

Comment:
```suggestion
    private lateinit var auth: FirebaseAuth
    private lateinit var googleSignInClient: GoogleSignInClient
```
cho vào AuthRepository